### PR TITLE
Fix invalid suggestion in error in apply_variable_ufunc

### PR DESCRIPTION
**Pull request by Illviljan**
The suggestion appears to be copy/paste-friendly but the format was slightly off.

### CODE REVIEW

**Comment by Illviljan** [comment]
Unrelated errors, see  #5366.

File: xarray/core/computation.py [hunk 1]
```diff
@@ -678,7 +678,7 @@ def apply_variable_ufunc(
                                     "apply_ufunc with dask='parallelized' consists of "
                                     "multiple chunks, but is also a core dimension. To "
                                     "fix, either rechunk into a single dask array chunk along "
-                                    f"this dimension, i.e., ``.chunk({dim}: -1)``, or "
+                                    f"this dimension, i.e., ``.chunk(dict({dim}=-1))``, or "
```
Comment (max-sixty) [inline xarray/core/computation.py:681]:
```suggestion
                                    f"this dimension, i.e., ``.chunk(dict(dim=-1))``, or "
```

Comment (Illviljan) [reply xarray/core/computation.py:681]: But then it's not copy/paste friendly? 
I got this error a few times in the cross product pr and I felt it was a nice and helpful idea to just be able to append `.chunk(dict(cartesian=-1))` without having to think.

Comment (max-sixty) [reply xarray/core/computation.py:681]: Sorry! I misread this. Your idea is great

**Comment by max-sixty** [comment]
Thanks @Illviljan !

File: xarray/core/computation.py (hunk 1 above)
Comment (keewis) [reply xarray/core/computation.py:681]: note that this is not copy-paste friendly in general: `dict(1=-1)` is invalid python

**Pull request merged**